src/app/auth/components/register/register.component.html

```
@@ -23,6 +22,8 @@ <h2 class="text-2xl md:text-3xl font-bold text-center mb-6">
           (formSubmit)="onFormSubmit($event)"
           class="w-full">
         </app-dynamic-form>
+        <p class="text-center text-sm text-gray-600 mt-4">
+          Already have an account? <a (click)="goToLogin()" class="text-blue-500 hover:underline cursor-pointer">Login</a>
```

HTMLHint (1.5.0)

[warning] The id and class attribute values must be in lowercase and split by a dash.

(id-class-value)